Dockerfile

```diff
@@ -11,4 +11,17 @@ RUN GRADLE_USER_HOME=cache ./gradlew buildDeb -x test && \
   cd .. && \
   rm -rf workdir
 
+RUN apt-get -y update && \
```

Comment:
I suggest merging this into the previous `RUN` command. If you download it to `workdir`, and unzip it to `workdir/gcloud`, you won't have worry about removing the `.zip` file or its inflated contents, because it will get wiped away with `rm -rf  workdir`. As it is currently, you've left behind the `/builder` dir.